spec/forms/registration_form_spec.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'byebug'
+
+RSpec.describe RegistrationForm do
+  describe '#save' do
+    application ||= Fabricate(:application)
+
+    context 'given valid user input' do
+      params = {
+        email: "test+#{Time.now.to_i}@nimblehq.co",
+        password: '123456',
+        password_confirmation: '123456',
+        client_id: application.uid
+      }
+      it 'returns true' do
+        form = described_class.new
+
+        expect(form.save(params)).to be(true)
+      end
+    end
```

Comment:
The fixtures should be located within the `it ...` like the example below 

```suggestion
RSpec.describe RegistrationForm do
  describe '#save' do
    context 'given valid user input' do
      it 'returns true' do
        params = {
          email: "test+#{Time.now.to_i}@nimblehq.co",
          password: '123456',
          password_confirmation: '123456',
          client_id: application.uid
        }

        form = described_class.new

        expect(form.save(params)).to be(true)
      end
    end
```